File: Assets/Smash Way/Scripts JS/GameOther/TriggerOther.js
#pragma strict

var ThisRotation:boolean=false;
var Cam:GameObject;
var Angle:int;
var AngleSpeed:float;

function OnTriggerEnter(other:Collider){
if(other.gameObject.tag=="Camera"){
if(ThisRotation){
Cam.GetComponent.<CamAnim>().ReservRotate=Angle;
Cam.GetComponent.<CamAnim>().RotateSpeed=AngleSpeed;
Cam.GetComponent.<CamAnim>().Rotate();
}
}
}

File: Assets/Smash Way/Scripts CS/GameOther/TriggerOther.cs
using UnityEngine;
using System.Collections;

public class TriggerOther : MonoBehaviour {



	public bool ThisRotation=false;
	public GameObject Cam;
	public int Angle;
	public float AngleSpeed;

	void OnTriggerEnter(Collider other){
		if(other.gameObject.tag=="Camera"){
			if(ThisRotation){
				Cam.GetComponent<CamAnim>().ReservRotate=Angle;
				Cam.GetComponent<CamAnim>().RotateSpeed=AngleSpeed;
				Cam.GetComponent<CamAnim>().Rotate();
			}
		}
	}
}
